biome.jsonc

```diff
@@ -15,9 +15,21 @@
     },
     "rules": {
       "recommended": true,
-      "nursery": {
-        "useJsonImportAttribute": "error",
+      "correctness": {
+        "useJsonImportAttributes": "error",
```

Comment:
i think we should add this to our ESM config (if not the main one)

Reply:
Oh this is actually part of the `project` domain now and is enabled by default in all of our shared configs. Will remove this.